src/test/kotlin/racing/CarKoTest.kt

```diff
@@ -0,0 +1,39 @@
+package racing
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.data.forAll
+import io.kotest.data.row
+import io.kotest.matchers.shouldBe
+
+class CarKoTest : DescribeSpec({
+    describe("move method") {
+        val moveThreshold = 10
+
+        forAll(
+            row(moveThreshold),
+            row(moveThreshold + 4),
+            row(moveThreshold + 10)
+        ) { power ->
+            context("with power : $power") {
+                val car = Car(moveThreshold = moveThreshold)
+                car.move(power)
+                it("move forward when power greater or equals than threshold") {
+                    car.position shouldBe 1
+                }
+            }
```

Comment:
context ~ it 사이에 구현이 있어 개인적으로 가독성이 좀 떨어진다고 느낍니다. 이런형식은 어떠신지요.
```suggestion
            context("with power : $power") {
                it("move forward when power greater or equals than threshold") {
                    with(Car(moveThreshold = moveThreshold)) {
                        move(power)
                        position shouldBe 1
                    }
                }
```

Reply:
저도 이 부분 작성하면서 초기화의 위치를 고민을 했는데 확실히 제안주신 코드가 가독성 좋아보이네요.

저는 dci 에서 c 가 "테스트 대상이 놓인 상황을 설명한다." 라고 하여 
"context 에서 초기화 -> 하나의 상황으로 다양한 it 을 테스트" 라 생각하여 context 에서 초기화하는 결정을 했습니다.

Reply:
👍